cartridges/int_cloudinary/cartridge/scripts/util/cloudinaryConstants.js

```diff
@@ -188,7 +188,7 @@ CloudinaryConstants.CLD_APPLY_CONTENT_VIDEO_ABS_URL_TRANSFORM_ERROR = 'Error occ
 CloudinaryConstants.CLD_APPLY_CONTENT_REL_URL_TRANSFORM_ERROR = 'Error occured while applying transformations on static library content absolute URL, URL : {0}, error : {1}';
 CloudinaryConstants.CLD_GET_SMD_URL_ERROR = 'Error occured while building product image URL base on SMD, product ID: {0}, error : {1}';
 CloudinaryConstants.CLD_GET_PRODUCT_PRIMARY_IMG_URL_ERROR = 'Error occured while building product primary image url using tag name, product ID: {0}, error : {1}';
-CloudinaryConstants.CLD_GET_PRODUCT_IMG_URL_BY_POSITION_ERROR = 'Error occured while building product image url by postion using tag name, product ID: {0}, error : {1}';
+CloudinaryConstants.CLD_GET_PRODUCT_IMG_URL_BY_POSITION_ERROR = 'Error occured while building product image url by Position using tag name, product ID: {0}, error : {1}';
```

Comment:
small letters, please

Reply:
@asad-rafter please put 'position' instead of 'Position'

Reply:
its outdated, it has been fixed